modules/402-ingress-nginx/crds/ingress-nginx.yaml

```diff
@@ -43,6 +43,7 @@ spec:
                     The way traffic goes to cluster from the outer network.
                     * `LoadBalancer` — Ingress controller is deployed and the service of `LoadBalancer` type is provisioned.
                     * `LoadBalancerWithProxyProtocol` — Ingress controller is deployed and the service of `LoadBalancer` type is provisioned. Ingress controller uses proxy-protocol to get a real IP of the client.
+                    * `L2LoadBalancer` - Ingress controller is deployed and the services of `LoadBalancer` (MetalLB) type is provisioned in resilient mode.
```

Comment:
Also, we have to enrich the [examples](https://deckhouse.io/documentation/v1/modules/402-ingress-nginx/examples.html) page.